Composer/packages/electron-server/src/electronWindow.ts

```diff
@@ -46,6 +46,7 @@ export default class ElectronWindow {
     }
     this._currentBrowserWindow = new BrowserWindow(browserWindowOptions);
     this._currentBrowserWindow.on('page-title-updated', (ev) => ev.preventDefault()); // preserve explicit window title
+    this._currentBrowserWindow.webContents.on('new-window', this.onOpenNewWindow.bind(this));
```

Comment:
Do we want to bind the function outside and pass the bound function to this event?

Reply:
Would there be any benefit to doing that?

After reading [the docs for `on(event, listener)`](https://nodejs.org/api/events.html#events_emitter_on_eventname_listener) it sounds like it just adds the listener to the array of listeners for that event.
> Adds the listener function to the end of the listeners array for the event named eventName.

I don't think the bound function gets instantiated every time the event is fired.

Reply:
I was hoping that would be the case. We are good with it then. In that case there wouldn't be any benefit.